### DISCUSSION TITLE
Suggestion for forms in sample docs

### DISCUSSION THREAD

**Discussion by guiseek** [picocss/pico#353] [Show and tell]
When the input element is inside the label element, there is no need to make a reference with the "for" attribute in the label and, consequently, there is no need for the "id" attribute in the input.

the same thing goes for the other examples below where the input is inside the label

[picocss.com/docs/forms](https://picocss.com/docs/forms.html#forms)

```diff
    <!-- Markup example 1: input is inside label -->
-    <label for="firstname">
+    <label>
      First name
-      <input type="text" id="firstname" name="firstname" placeholder="First name" required>
+      <input type="text" name="firstname" placeholder="First name" required>
    </label>

-    <label for="lastname">
+    <label>
      Last name
-      <input type="text" id="lastname" name="lastname" placeholder="Last name" required>
+      <input type="text" name="lastname" placeholder="Last name" required>
    </label>
```

The examples are cleaner :)

**Comment by lucaslarroche** [comment]
Please have a look at the v2 documentation:
https://v2.picocss.com/docs/forms

Feedback is welcome.
